libs/components/validation/src/lib/modules/email-validation/fixtures/email-validation.component.fixture.html

```diff
@@ -8,7 +8,7 @@
     *ngIf="email.errors && email.errors.skyEmail && (email.dirty || email.touched)"
   >
     <div [hidden]="!email.errors.skyEmail.invalid">
-      Please enter valid email with format joe@abc.com
+      Please enter valid email with format joe&#64;abc.com
```

Comment:
I'm seeing a few of these; what's the background on the change?

Reply:
The new template syntax uses `@` as a control character, so Angular has a schematic to escape them. I believe this change is optional.